packages/riipen-ui/src/components/Form.test.jsx

```diff
@@ -0,0 +1,161 @@
+import { mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import React from "react";
+
+import Form from "./Form";
+import Typography from "./Typography";
+
+describe("<Form>", () => {
+  it("renders without errors", () => {
+    let error;
+
+    try {
+      mount(<Form />);
+    } catch (theError) {
+      error = theError;
+    }
+
+    expect(error).toEqual(undefined);
+  });
+
+  it("renders correct snapshot", () => {
+    const wrapper = mount(<Form />);
+
+    expect(toJson(wrapper)).toMatchSnapshot();
+  });
+
+  describe("default props", () => {
+    it("sets correct default props", () => {
+      const wrapper = mount(<Form />);
+
+      expect(wrapper.find("Form").props().enter).toEqual(true);
+      expect(wrapper.find("Form").props().errorScroll).toEqual(true);
+    });
+  });
+
+  describe("children prop", () => {
+    it("displays given children", () => {
+      const child = <div>child</div>;
+
+      const wrapper = mount(<Form>{child}</Form>);
+
+      expect(wrapper.containsMatchingElement(child)).toBeTruthy();
+    });
+  });
+
+  describe("classes prop", () => {
+    it("sets custom class", () => {
+      const classes = ["newClass"];
+
+      const wrapper = mount(<Form classes={classes} />);
+
+      expect(
+        wrapper
+          .find("Form")
+          .childAt(0)
+          .hasClass(classes[0])
+      ).toEqual(true);
+    });
+
+    it("appends higher order values to classes prop with withClass decorator", () => {
+      const classes = ["riipen", "riipen-form"];
+
+      const wrapper = mount(<Form />);
+
+      expect(wrapper.find("Form").props().classes).toEqual(classes.sort());
+    });
+  });
+
+  describe("error prop", () => {
```

Comment:
Can we check that the error gets displayed using `.containsMatchingElement()` or something similar? And maybe be check that it gets put inside a `Typography` element if it is a string.

Reply:
Added both plus a test for `renderErrorNode`